ui/docker/Dockerfile.run

```diff
@@ -0,0 +1,15 @@
+FROM nginx
+
+ENV LC_ALL=en_US.UTF-8 \
+    LANG=en_US.UTF-8
+
+RUN apt update
+RUN apt install -y vim
```

Comment:
Not quite sure you need this in a webserver environment :confused:

Reply:
docker folder is outdated, I deleted this folder. For the build part here is the PR : #782